<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanFacadeClass</class>
 <widget class="QMainWindow" name="ScanFacadeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScanFacade</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QListView" name="listView">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="cleanButton">
            <property name="text">
             <string>Clear Logging</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Data Source Manger(&amp;M)</string>
    </property>
    <widget class="QMenu" name="menuDS">
     <property name="title">
      <string>DS</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionload"/>
    <addaction name="actionopen"/>
    <addaction name="actionaquire_DS_list"/>
    <addaction name="separator"/>
    <addaction name="menuDS"/>
    <addaction name="separator"/>
    <addaction name="actionunload"/>
    <addaction name="actionclose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Date Source(&amp;D)</string>
    </property>
    <addaction name="actionopen_2"/>
    <addaction name="action_show_setup_dialog"/>
    <addaction name="separator"/>
    <addaction name="actionclose_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Scan Image(&amp;N)</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>logging(&amp;L)</string>
     </property>
     <addaction name="actionwindow_2"/>
     <addaction name="actionfile"/>
     <addaction name="actiondisable"/>
    </widget>
    <addaction name="actionStartScan"/>
    <addaction name="separator"/>
    <addaction name="actionprevious"/>
    <addaction name="actionnext"/>
    <addaction name="separator"/>
    <addaction name="actionInsert_Extenal_Image_I"/>
    <addaction name="actioninsert"/>
    <addaction name="actionReplace"/>
    <addaction name="actionDelete"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionpackage"/>
    <addaction name="actionUpload"/>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server(&amp;S)</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Chrome">
    <property name="title">
     <string>Chrome(&amp;C)</string>
    </property>
    <addaction name="actionMatchers"/>
   </widget>
   <addaction name="menuServer"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_Chrome"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionload">
   <property name="text">
    <string>Load DSM(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>连接数据源管理器</string>
   </property>
  </action>
  <action name="actionunload">
   <property name="text">
    <string>Unload DSM(&amp;U)</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>Open DSM(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开数据源管理器</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>Close DSM(&amp;C)</string>
   </property>
  </action>
  <action name="actionaquire_DS_list">
   <property name="text">
    <string>Aquire DS list(&amp;D)</string>
   </property>
   <property name="toolTip">
    <string>获得数据源列表</string>
   </property>
  </action>
  <action name="actionopen_2">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开数据源</string>
   </property>
  </action>
  <action name="actionclose_2">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_show_setup_dialog">
   <property name="text">
    <string>Show setup dialog(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>显示数据源的配置对话框</string>
   </property>
  </action>
  <action name="actionStartScan">
   <property name="text">
    <string>Start Scan..(&amp;S)</string>
   </property>
  </action>
  <action name="actionprevious">
   <property name="text">
    <string>Previous(&amp;&lt;)</string>
   </property>
  </action>
  <action name="actionnext">
   <property name="text">
    <string>Next(&amp;&gt;)</string>
   </property>
  </action>
  <action name="actioninsert">
   <property name="text">
    <string>Insert(&amp;I)</string>
   </property>
  </action>
  <action name="actionpackage">
   <property name="text">
    <string>Package(&amp;P)...</string>
   </property>
   <property name="toolTip">
    <string>打包压缩的图片</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="text">
    <string>Upload(&amp;U)</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear(&amp;C)</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace(R)</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete(&amp;D)</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start(&amp;S)</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop(&amp;T)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
  </action>
  <action name="actionInsert_Extenal_Image_I">
   <property name="text">
    <string>Insert Extenal Image(&amp;I)</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>文件</string>
   </property>
  </action>
  <action name="actionwindow">
   <property name="text">
    <string>window</string>
   </property>
  </action>
  <action name="actionwindow_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Window</string>
   </property>
  </action>
  <action name="actionfile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actiondisable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </action>
  <action name="actionMatchers">
   <property name="text">
    <string>Matches(&amp;M)...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ScanFacade.qrc"/>
 </resources>
 <connections/>
</ui>
